htdocs/xoops_lib/Xoops.php

```diff
@@ -733,7 +733,7 @@ public function isAdmin()
      *
      * @param boolean $optional true if failure to load handler should be considered a warning, not an error
      *
-     * @return \Xoops\Core\Kernel\Handlers\XoopsBlockHandler
+     * @return \Xoops\Core\Kernel\Handlers\XoopsBlockHandler|\Xoops\Core\Kernel\XoopsPersistableObjectHandler
```

Comment:
I understand that, but my phpstorm complains! I don't think this problem could be solved in the factory. We would have to add all possible classes the factory could produce in the @return doc. Ok, leave it as it is.